src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs

```diff
@@ -444,6 +456,7 @@ private void HandleUpDownKeys(KeyRoutedEventArgs e)
 			}
 
 			_suggestionsList.SelectedIndex = nextIndex;
+			_suggestionsList.ScrollIntoView(_suggestionsList.SelectedItem);
```

Comment:
It feels like this shouldn't be specific to up/down keys, but should happen whenever `_suggestionsList.SelectedItem` changes? i.e, listen to `_suggestionsList.SelectionChanged`?

Reply:
Good catch. I tested on WinUI and it indeed scrolling into view even with an explicit call to ListView.set_SelectedIndex.